build/ProcessJarFiles/JarProcessing.targets

```diff
@@ -58,6 +61,16 @@
     <Exec Command="java -jar $(RuleExtractorJar) --plugins $(JarDownloadDir)\%(PluginJars.JarName) --languages=%(PluginJars.LanguageKey) --output=$(_WorkingDirectory)\%(PluginJars.LanguageKey)-rules.json" />
   </Target>
 
+
+  <Target Name="ExtractRuleDescFromJson">
+    <Message Importance="high" Text="[ProcessJarFiles] Checking for existence of rule extractor exe: $(RuleDescExtractorExe) ..." />
+    <Error Condition="!Exists($(RuleDescExtractorExe))" Text="Rule description extractor exe does not exist: $(RuleDescExtractorExe)" />
+    <Message Importance="high" Text="[ProcessJarFiles]    ... exe exists." />
+
+    <!-- TODO - call exe -->
```

Comment:
So far the targets just check that the exe has been built and can be found.

Actually calling the exe will be done in a separate PR.